server/unithand.cpp

```diff
@@ -3772,6 +3761,24 @@ static bool unit_bombard(struct unit *punit, struct tile *ptile,
   }
   unit_list_iterate_safe_end;
 
+  // Notify the players hit by the bombardment
+  for (auto player_to_notify : players_to_notify) {
+    notify_player(
+        player_to_notify, ptile, E_UNIT_BOMB_DEF, ftc_server,
+        /* TRANS: Your units in [tile] have been bombarded by the French
+           Bomber.*/
+        _("Your units in %s were bombarded by a %s %s %s [id:%d]."),
+        tile_link(ptile), nation_adjective_for_player(pplayer),
+        unit_veteran_level_string(punit), unit_name_translation(punit),
+        punit->id);
+  }
+  // Notify the player that bombarded
+  notify_player(pplayer, ptile, E_UNIT_BOMB_ATT, ftc_server,
+                /* TRANS: Your Bomber bombarded [coords?].*/
+                _("Your %s %s [id:%d] bombarded the units at %s."),
```

Comment:
or... do the other two above as 'at' for the tile coords?

Reply:
changed to "at"